Confus/Confus/Game.cpp

```diff
@@ -147,6 +147,24 @@ namespace Confus
         }
     }
 
+    void Game::updateOtherPlayers(/*array a_PlayerInfo*/)
```

Comment:
Because I started working on that function before the callback map was made, I will update it in the future.